### PR TITLE
Dtspo 8041 Tag policy nonprod assignments

### PR DESCRIPTION
**Pull request by Jenniferstrej**
### Change description ###
HMCTS Tagging policy assignments for all non-prod environments:

- Preview
- Perftest 
- ITHC
- DEMO 
- PTL
- AAT

NOTE: All assignments have DoNotForce enforcement mode given we are still working on fixing tags (see epic https://tools.hmcts.net/jira/browse/DTSPO-7631)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: assignments/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/assign.tagging.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/dts002/providers/Microsoft.Authorization/policyDefinitions/HMCTSTagging",
+      "scope": "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
+    },
+    "sku": {
+      "name": "A0",
+      "tier": "Free"
+    },
+    "type": "Microsoft.Authorization/policyAssignments",
+    "id": "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/providers/Microsoft.Authorization/policyAssignments/HMCTSTagging_DTS-CFTPTL-INTSVC",
+    "name": "HMCTSTagging_DTS-CFTPTL-INTSVC"
```
Comment (darren221980) [inline assignments/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/assign.tagging.json:30]: Looking at this and the other assignments below and thinking, should the name include the subscription name or just the environment name. In this case the subscription DTS-CFTPTL-INTSVC is referred to as PTL and is commonly known as that so makes sense to use this to me.

be interesting to see if others agree here or not.

Comment (Jenniferstrej) [reply assignments/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/assign.tagging.json:30]: I thought about that but kept following the other assignments that had the actual subscription name appended to the name and ID. I did try to put the names we generally refer to in the "Display Name" field. But I'm happy to change if most think it should be reflected on the name too.

Comment (thomast1906) [reply assignments/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/assign.tagging.json:28]: I think this is fine as displayName `HMCTS Tagging - CFT PTL` shows a friendly reader name

File: assignments/subscriptions/1497c3d7-ab6d-4bb7-8a10-b51d03189ee3/assign.tagging.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to check required tags",
+      "displayName": "HMCTS Tagging - CFT PTL (DTS-CFTSBOX-INTSVC)",
```
Comment (thomast1906) [inline assignments/subscriptions/1497c3d7-ab6d-4bb7-8a10-b51d03189ee3/assign.tagging.json:7]: Can we include `CFT PTL SBOX`

Comment (Jenniferstrej) [inline assignments/subscriptions/1497c3d7-ab6d-4bb7-8a10-b51d03189ee3/assign.tagging.json:7]:
```suggestion
      "displayName": "HMCTS Tagging - CFT PTL SBOX (DTS-CFTSBOX-INTSVC)",
```

**Review by darren221980 — APPROVED** [review]

**Pull request merged**